packages/karma/package.json

```diff
@@ -25,7 +25,7 @@
     "karma": "^4.0.0",
     "karma-chrome-launcher": "2.2.0",
     "karma-firefox-launcher": "1.1.0",
-    "karma-jasmine": "1.1.1",
+    "karma-jasmine": "2.0.1",
```

Comment:
is this version bump required in this PR? just curious

Reply:
No, this version bump is not required in this PR. I've reverted this change, will create another PR just for this.